<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подключение</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="connectionNameLabel">
       <property name="text">
        <string>Название подключения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="connectionNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="dbGroupBox">
     <property name="title">
      <string>База данных</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="sqliteRadioButton">
          <property name="toolTip">
           <string>Субд sqlite встроена в данную программу</string>
          </property>
          <property name="text">
           <string>sqlite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mysqlRadioButton">
          <property name="toolTip">
           <string>Субд MySQL устанавливается отдельно от данной программы</string>
          </property>
          <property name="text">
           <string>MySQL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="postgresqlRadioButton">
          <property name="text">
           <string>PostgreSQL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="dbhostnameLabel">
          <property name="text">
           <string>Имя хоста базы данных:</string>
          </property>
          <property name="buddy">
           <cstring>dbhostnameLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dbhostnameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="dbusernameLabel">
          <property name="text">
           <string>Логин:</string>
          </property>
          <property name="buddy">
           <cstring>dbusernameLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dbusernameLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="dbpasswordLabel">
          <property name="text">
           <string>Пароль:</string>
          </property>
          <property name="buddy">
           <cstring>dbpasswordLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dbpasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QLabel" name="databasenameLabel">
          <property name="text">
           <string>Имя базы данных:</string>
          </property>
          <property name="buddy">
           <cstring>databasenameLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="databasenameLineEdit">
          <property name="toolTip">
           <string>Для sqlite необходимо указать полный путь и имя файла базы данных . Для MySQL это название схемы.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createTablesPushButton">
          <property name="toolTip">
           <string>Создает в новой базе данных таблицы, необходимые для работы программы. Таблицы необходимо создавать единократно, после создания новой базы данных.</string>
          </property>
          <property name="text">
           <string>Создать таблицы</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="movePerformersPushButton">
       <property name="text">
        <string>Переместить исполнителей</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="../kcrm.qrc">
         <normaloff>:/images/ok32.png</normaloff>:/images/ok32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="icon">
        <iconset resource="../kcrm.qrc">
         <normaloff>:/images/cancel32.png</normaloff>:/images/cancel32.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../kcrm.qrc"/>
 </resources>
 <connections/>
</ui>
